src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -124,6 +133,18 @@ object Coroutine {
       } else throw new CoroutineStoppedException
     }
 
+    /** Calls `resume` until the either the coroutine yields a value or returns.
+      * If `pull` returns `true`, then the coroutine has suspended by yielding
```

Comment:
Please separate the one-liner description from the rest of the comment with an empty line.